### DISCUSSION TITLE
zksync local node deployment error , npx zksync-cli dev config

### DISCUSSION THREAD

**Discussion by DJ0301** [Cyfrin/foundry-full-course-cu#2129] [Q&A]
I have been trying to run a local node on my mac but I'm being faced with this error, I tried changing down the versions of cliBoxes so that it doesn't use the with syntax but unfortunately that doesn't seem to work , how can I tackle this issue?

npx zksync-cli dev config
file:///Users/dhananjayjoshi/.npm/_npx/ed2d5270b29717fa/node_modules/cli-boxes/index.js:1
import cliBoxes from './boxes.json' with {type: 'json'};
                                    ^^^^

SyntaxError: Unexpected token 'with'
    at DefaultModuleLoader.moduleStrategy (node:internal/modules/esm/translators:116:18)
    at DefaultModuleLoader.moduleProvider (node:internal/modules/esm/loader:190:14)

Node.js v20.4.0

**Comment by EngrPips** [comment]
Hello @DJ0301, I followed Patrick's instructions in this video `https://updraft.cyfrin.io/courses/foundry/foundry-simple-storage/zksync-local-node?lesson_format=video` of the course, and everything worked for me. Patrick stated that if you are having a hard time setting up zksync-local node, just move on as `Anvil` is enough for everything you will do.
